### PR TITLE
Added information about using Maven and Maven's directory layout for Java-based projects.

### PR DESCRIPTION
**Pull request by Belberus**
- What's Maven and Mavens directory layout for Java-based projects.
- What's the structure of our project and checking that our structure follows Mavens one.

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -60,3 +60,58 @@ SOME OTHER TASKS OF GRADLE
...
+		- src/it: Integration Tests (primarily for plugins).
+		- src/assembly:	Assembly descriptors.
+		- src/site:	Site.
+		- LICENSE.txt: Project's license.
+		- NOTICE.txt: Notices and attributions required by libraries that the project depends on.
+		- README.txt: Project's readme.
+
+	In our case, our project have the next structure:
+
+	|src ----- main ----- java ----- es ----- unizar ----- webeng ----- hello
```
Comment (RecuencoJones) [inline description.md]: Too much space used, try something like:

```
/
|--src/
|----main/
|------java/
|--------es/unizar/webeng/hello/
|------resources/
|------webapp/
|--------WEB-INF/
|----------web.xml
|----test/
|----java/
|------es/unizar/webeng/hello/
| // continue with all other files
```

File: description.md [hunk 2]
```diff
@@ -60,3 +60,58 @@ SOME OTHER TASKS OF GRADLE
...
+	|description.md
+	|
+	|build
+	|
+ 	|gradle
+ 	|
+ 	|other files
+
+
+ 	As we can see our project follows Mavens directory layout. There are some directories in the layout that aren't present in our project structure but this is because our project doesn't use anything to be included in those directories so there is no point on having them in our structure.
```
Comment (RecuencoJones) [inline description.md]: Worth to mention that some of those are generated on build time and that's why they should be excluded

File: description.md [hunk 3]
```diff
@@ -60,3 +60,58 @@ SOME OTHER TASKS OF GRADLE
...
+	Maven is a tool used for building and managing Java-based projects. The primary goal of Maven is to allow developers to comprehend the complete state of a development effort in the shortest period of time, Maven helps us:
+		- Making the build process easy.
+		- Providing a uniform build system.
+		- Providing quality project information.
+		- Providing guidelines for best practices development.
+		- Allowing trasnparent migration to new features.
+
+	MAVEN STANDARD DIRECTORY LAYOUT
+
+		Having a common directory layout help users familiar with one Maven project to don't get lost in another different Maven project. The directory layout created by Maven have the directories exposed right below and you should try to confort your structure to this one as much as possible:
```
Comment (RecuencoJones) [inline description.md]: to not get lost

**Comment by fjlopez** [comment]
I recommend to apply the comments from your fellow. Updates will added automatically to the pull request when you push changes to your master. In addition, sync your repo with the latest changes.

Add here a comment when the issues are fixed.

**Comment by Belberus** [comment]
I'm going to close the pull request, apply the changes and make a new one
because I'm actually making a mess and I think this way will be faster :)

2016-09-25 16:58 GMT+02:00 Francisco J Lopez-Pellicer <
notifications@github.com>:

**Pull request closed without merging**